src/templates/utils-common.js

```diff
@@ -214,6 +214,10 @@ export function setLocaleCookie (locale, res, { useCookie, cookieDomain, cookieK
     }
 
     const redirectCookie = cookieSerialize(cookieKey, locale, cookieOptions)
+    headers = headers.filter(header => {
+      const cookie = cookieParse(header)
+      return !cookie[cookieKey]
```

Comment:
I'd rather check whether the cookie name matches rather than checking if the cookie value is truthy:
```suggestion
      return cookieKey in cookie
```

Reply:
Also, parsed header can include more than one cookie so in theory this could unintentionally drop some irrelevant cookies.

Though if we assume that only this module sets the locale cookie then I guess it should always be set on its own (unless node does some merging on its own but I doubt)...

Reply:
Node does this merging staff, so I think it's not possible to have the same key in different cookies, don't worry about it.